lib/patch-series.ts

```diff
@@ -140,10 +140,12 @@ export class PatchSeries {
                 pullRequestURL,
             };
         } else {
-            if (!await git(["rev-list",
-                            `${metadata.headCommit}...${headCommit}`],
-                           { workDir })) {
-                throw new Error(`${headCommit} was already submitted`);
+            if (!options.noUpdate) {    // allow reprint of submitted PRs
```

Comment:
Good.

Two suggestions:

1. introduce the `noUpdate` field in its own, preparatory commit. That structures the commit history better, and makes the changes easier to review.
2. extend the condition in the already-existing `if`: 

   ```typescript
   if (!options.noUpdate &&
       !await git(["rev-list",
   [...]
   ```